dockerfiles/scripts/install_cran_pkgs.R

```diff
@@ -55,7 +55,9 @@ cran_pkgs <- list(
     "igraph",
     "dm",
     "glmmTMB",
-    "rstan"
+    "rstan",
+    "rtables",
```

Comment:
rtables is Roche package - shouldn't we take it from github directly ?

Reply:
`rtables` is published on CRAN: https://cran.r-project.org/web/packages/rtables/index.html